lab_5_scrapper/scrapper.py

```diff
@@ -90,75 +214,151 @@ def __init__(self, config: Config) -> None:
         """
         Initializes an instance of the Crawler class
         """
-        pass
+        self.config = config
+        self.urls = []
 
     def _extract_url(self, article_bs: BeautifulSoup) -> str:
         """
         Finds and retrieves URL from HTML
         """
-        pass
+        links = article_bs.find_all('a')
+        for link in links:
+            url = link.get('href')
+            if url and url.count('/') == 4 and url[:9] == '/novosti/':
```

Comment:
`url[:9] == '/novosti/'` should be reworked, I did not see this

Reply:
i added this conditions according to your previous comment "are there any more specific checks? maybe there is something special between these slashes? otherwise the check is too vague". besides, during one of our seminars you also pointed out the need to find some more patterns for extracting the needed urls except for the number of slashes. did i come up with a bad pattern or should i just implement it not using slicing but regex, for example?